javascript-challenges.test.js

```diff
@@ -14,30 +14,101 @@
 
 // a) Create a test with expect statements using the variables provided.
 
+describe("convertMessage", () => {
+
 const secretCodeWord1 = "Lackadaisical"
 // Expected output: "L4ck4d41s1c4l"
 const secretCodeWord2 = "Gobbledygook"
 // Expected output: "G0bbl3dyg00k"
 const secretCodeWord3 = "Eccentric"
-// Expected output: "3cc3ntr1c"
+// Expected output: "3cc3ntr1c"  
+
+  it("takes in a string and returns a string with a coded message", () => {
+    expect(convertMessage(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+    expect(convertMessage(secretCodeWord2)).toEqual("G0bbl3dyg00k")
+    expect(convertMessage(secretCodeWord3)).toEqual("3cc3ntr1c")
+  })
+})
+
+// -------> convertMessage is not defined
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//Create a function convertMessage
+//input: string
+//const codedMessage variable to store converted message
+//use .replace() to replace letters for numbers
+  // /a/ regular expression pattern that matches the charecter 'a'. 'g' modefier "global" that tells JavaScript to find and replace all occurrences of the pattern in the string. Without it it would only replace the first occurance.
+//output string with converted message
+
+const convertMessage = (string_input) => {
+  const codedMessage = string_input
```

Comment:
Excellent work on this problem!  Since you are already utilizing regex, you could handle case insensitive with that as well by adding an i modifier along with the global modifier. https://www.w3schools.com/jsref/jsref_regexp_i.asp